lib/CMakeLists.txt

```diff
@@ -127,6 +127,17 @@ endforeach(item ${QUDA_OBJS})
 
 list(REMOVE_ITEM QUDA_OBJS ${QUDA_CU_OBJS})
 
+set(QUDA_MDW_FUSED_LS_LIST "4,8,12,16,20,24" CACHE STRING "The list of Ls the get compiled")
```

Comment:
Typo: The list of Ls that get compiled

I might make it a little clearer that it only applies to the fused operator, that is, List of Ls values that are instantiated for the fused Mobius operator

Reply:
No need to make it clearer I think with respect to which operator it applies to, since this PR is a stepping stone to broader tensor-core acceleration for M5 inverse across the board which will have this same behaviour.

Reply:
Fair enough---I'd still like the typo fixed 😄

Reply:
https://github.com/lattice/quda/pull/1224/commits/c0f74b86e8bc0a7d350aa5f51fb64775d3d09986